MTM-52611 add alarms redesign announcement (#1877)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md

---
date: 2024-03-28
title: Alarms redesign and improvements
product_area: Application enablement & solutions
change_type:
  - value: change-inv-3bw8e
    label: Announcement
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-52611
version: 1020.0.0
---

> In a future version, we are introducing redesigned alarms with both visual and functional enhancements.

Reviewer: ```suggestion
In a future version, we will introduce redesigned alarms with both visual and functional enhancements.
```

> **Visual and UX Enhancements**:

Reviewer: ```suggestion
**Visual and UX enhancements**:
```

* We have redesigned the way alarms are displayed in the "Alarms" view and widgets.

> **Functional Improvements**:

Reviewer: ```suggestion
**Functional improvements**:
```

> * The "Alarms" view now shows only a single list of alarms, where filters can be applied.

Reviewer: ```suggestion
* The "Alarms" view now only shows one single list of alarms, in which filters can be applied.
```

> * We have introduced a details section, which contains all information related to the alarm.

Reviewer: ```suggestion
* A details section has been introduced which contains all information related to the alarm.
```

* The functionalities of the "Alarm list", "All critical alarms", and "Recent alarms" widgets have been merged into a single, comprehensive widget called "Alarm list".

> * The Alarm list widget has been migrated to Angular.

Reviewer: ```suggestion
* The "Alarm list" widget has been migrated to Angular.
```

> * Real-time functionality has been replaced with Auto refresh.

Reviewer: ```suggestion
* The real-time functionality has been replaced with an auto-refresh functionality.
```

---
date: 2024-03-28
title: Alarms redesign and improvements
product_area: Application enablement & solutions
change_type:
  - value: change-inv-3bw8e
    label: Announcement
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-52611
version: 1020.0.0
---

> In the upcoming version, we will introduce redesigned alarms with both visual and functional enhancements.

Reviewer: ```suggestion
In the upcoming version, a redesign of the alarms feature will be introduced with both visual and functional enhancements.
```

**Visual and UX enhancements**:

> * We have redesigned the way alarms are displayed in the "Alarms" view and widgets.

Reviewer: ```suggestion
* The way alarms are displayed in the "Alarms" view and widgets has been redesigned.
```
